tests/Resources/ti.ui.tabgroup.test.js

```diff
@@ -0,0 +1,249 @@
+		});
+		tab = Ti.UI.createTab({
+			title: 'Tab',
+			window: win
+		});
+
+		tabGroup.addTab(tab);
+		should(tabGroup.allowUserCustomization).eql(true);
+		tabGroup.setAllowUserCustomization(false);
+		should(tabGroup.allowUserCustomization).eql(false);
+	});
+
+	it.ios('.tabsTranslucent', () => {
+		var win = Ti.UI.createWindow();
+		tabGroup = Ti.UI.createTabGroup({
+			tabsTranslucent: true
+		});
+		tab = Ti.UI.createTab({
+			title: 'Tab',
+			window: win
+		});
+
+		tabGroup.addTab(tab);
+		should(tabGroup.tabsTranslucent).eql(true);
+		tabGroup.setTabsTranslucent(false);
+		should(tabGroup.tabsTranslucent).eql(false);
+	});
+
+	it('#setTabs()', () => {
+		const winA = Ti.UI.createWindow(),
+			tabA = Ti.UI.createTab({
+				title: 'Tab A',
+				window: winA
+			}),
+			winB = Ti.UI.createWindow(),
+			tabB = Ti.UI.createTab({
+				title: 'Tab B',
+				window: winB
+			});
+		tabGroup = Ti.UI.createTabGroup();
+
+		tabGroup.setTabs([ tabA, tabB ]);
+		should(tabGroup.getTabs()).eql([ tabA, tabB ]);
+	});
+
+	it.windowsBroken('#setActiveTab()', finish => {
+		const winA = Ti.UI.createWindow(),
+			tabA = Ti.UI.createTab({
+				title: 'Tab A',
+				window: winA
+			}),
+			winB = Ti.UI.createWindow(),
+			tabB = Ti.UI.createTab({
+				title: 'Tab B',
+				window: winB
+			});
+		tabGroup = Ti.UI.createTabGroup();
+
+		// Does windows fire this event?
+		// Can we test this without even opening tab group?
+		tabGroup.addEventListener('open', () => {
+			try {
+				tabGroup.setActiveTab(tabB);
+				should(tabGroup.getActiveTab().title).be.a.String;
+				should(tabGroup.getActiveTab().title).eql('Tab B');
+				finish();
+			} catch (err) {
+				finish(err);
+			} finally {
+				tabGroup.removeTab(tabA);
+				tabGroup.removeTab(tabB);
+			}
+		});
+
+		tabGroup.addTab(tabA);
+		tabGroup.addTab(tabB);
+		tabGroup.open();
+	});
+
+	it.android('#disableTabNavigation()', function (finish) {
+		var winA = Ti.UI.createWindow(),
+			tabA = Ti.UI.createTab({
+				title: 'Tab A',
+				window: winA
+			}),
+			winB = Ti.UI.createWindow(),
+			tabB = Ti.UI.createTab({
+				title: 'Tab B',
+				window: winB
+			});
+		this.timeout(5000);
+		tabGroup = Ti.UI.createTabGroup();
+
+		// does windows fire this event?
+		tabGroup.addEventListener('open', () => {
+			try {
+				tabGroup.disableTabNavigation(true);
+				tabGroup.setActiveTab(tabB);
+				should(tabGroup.getActiveTab().title).be.a.String;
+				should(tabGroup.getActiveTab().title).eql('Tab A');
+				tabGroup.disableTabNavigation(false);
+				tabGroup.setActiveTab(tabB);
+				should(tabGroup.getActiveTab().title).be.a.String;
+				should(tabGroup.getActiveTab().title).eql('Tab B');
+				finish();
+			} catch (err) {
+				finish(err);
+			} finally {
+				tabGroup.removeTab(tabA);
+				tabGroup.removeTab(tabB);
+			}
+		});
+
+		tabGroup.addTab(tabA);
+		tabGroup.addTab(tabB);
+		tabGroup.open();
+	});
+
+	it('.title', () => {
+		tabGroup = Ti.UI.createTabGroup({
+			title: 'My title'
+		});
```

Comment:
I don't think iOS really uses this. On iOS, the title is managed per window-in-tab instance.